pkg/virt-operator/kubevirt.go

```diff
@@ -1151,3 +1152,32 @@ func (c *KubeVirtController) syncDeletion(kv *v1.KubeVirt) error {
 	logger.Info("Processed deletion for this round")
 	return nil
 }
+
+// checkIfConfigMapStillExists This function validates that the obsolete kubevirt-config configMap is no longer exist.
+// The user should not use this configMap to configure KubeVirt, and KubeVirt will ignore it anyway. In case the configMap
+// still exists, this function emit an event to notify the user, and ask them to delete it.
+func (c *KubeVirtController) checkIfConfigMapStillExists(logger *log.FilteredLogger, trigger <-chan time.Time) {
```

Comment:
This ticker logic seems to be slihtly over-engineered. how about using `wait.Until`:


```go
	wait.Until(func() {
		logger.V(5).Info("read the kubevirt-config configMap. if exist, emmit an event")
		cm, err := c.clientset.CoreV1().ConfigMaps(c.operatorNamespace).Get(ctx, obsoleteCmName, getOpts)
		if err != nil {
			if errors.IsNotFound(err) {
				logger.V(5).Info("The obsolete kubevirt-config configMap could not be found. good.")
			} else {
				logger.Errorf("can't get the kubevirt-config configMap %v", err)
			}
		}
		logger.Warning("The obsolete kubevirt-config configMap still exists. Please remove it.")
		c.recorder.Eventf(cm, k8sv1.EventTypeWarning, obsoleteCMReason, msg)
	}, time.Minute * 10, stopChan)
```

Reply:
I don't have a strong opinion about it, but as far as I understand it, ticker is the "standard" golang way to do cron-like tasks.

Reply:
Yes, just make your life a little bit easier and use `wait.Util` (btw. check out the wait file, there are really nice nuances of ticker based actions implemented.

Reply:
done